### ISSUE TITLE
Update the version of org.asciidoctor:asciidoctorj-groovy-dsl used to one that is published to Maven Central

### ISSUE THREAD

**Issue by erdi** [asciidoctor/asciidoctor-gradle-plugin#620]
The latest published version of the plugin, 3.3.2, resolves `org/asciidoctor/asciidoctorj-groovy-dsl:2.0.0` when applied to a build. That version of `org.asciidoctor:asciidoctorj-groovy-dsl` is only available from JCenter which will be sunset in 2022 which in turn will break people's builds using this plugin on machines which have not resolved the artifact from JCenter previously. 

There are [versions of this artifact available from Maven Central](https://search.maven.org/artifact/org.asciidoctor/asciidoctorj-groovy-dsl) - it's probably a good idea to update to using one of them given that users of this plugin are unable to configure the version of it used due to it being resolved via a detached configuration.

**Comment by mrotteveel** [comment]
A workaround is to add this to your build.gradle (or extend your current asciidoctorj config)
```
asciidoctorj {
    modules {
        groovyDsl {
            version '2.0.2'
        }
    }
}
```

**Comment by erdi** [comment]
Awesome workaround, thanks for sharing @mrotteveel! [Bye bye, JCenter](https://github.com/geb/geb/commit/7e2343d3dd76e2630dbd58a1a5b30c168be08c33)! Out of curiosity - how did you come up with it? Did you find the idea for it in the docs somewhere or did you have to resort to looking through the source?

**Comment by mrotteveel** [comment]
@erdi I already did something similar for the PDF plugin, guessed there was a similar mechanism for the groovy-dsl, and then checked the source. However, it is also documented at https://asciidoctor.github.io/asciidoctor-gradle-plugin/master/user-guide/#asciidoctorj-modules

**Comment by erdi** [comment]
I see. Thanks for explaining, @mrotteveel. Looks like I tried doing it the hard way and even contributed a fix when a simple workaround was already available. 🤦‍♂️

**Comment by ysb33r** [comment]
Permanently fixed in 4.0.0-alpha.1
